query/expression.go

```diff
@@ -23,6 +23,55 @@ import (
 	"github.com/square/metrics/query/aggregate"
 )
 
+const simultaneousFetchLimit = 4
```

Comment:
it should be some option that can be controllable per app. we can pair on it later to extract it out correctly.